mcs/build/tests.make

```diff
@@ -157,6 +157,9 @@ $(test_makefrag): $(test_response)
 
 -include $(test_makefrag)
 
+build-test-lib: $(test_lib)
```

Comment:
@kumpera FYI this is already supported via `make test PROFILE=net_4_x` <-- only builds the test lib and doesn't run it.